dsl/pact.go

```diff
@@ -69,6 +70,9 @@ type Pact struct {
 	// Examples include 'tcp', 'tcp4', 'tcp6'
 	// Defaults to 'tcp'
 	Network string
+
+	// Ports MockServer can be deployed to, can be CSV or Range with a dash
+	AllowedMockServerPorts string // Example "1234", "12324,5667", "1234-5667"
```

Comment:
Can you please move the `// Example...` bit above the field definition? (unless I am missing something with the way documentation is generated and this is somehow better)